### PR TITLE
Notebook Updates for iBSim2022 - CIL Version 22.1.0

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
This branch was used to run the iBSim training.

- Add Seeds in the Box Exercise 01 (https://github.com/TomographicImaging/CIL-Demos/pull/102)

- Seeds in the Box Exercise 02 (https://github.com/TomographicImaging/CIL-Demos/pull/103)

-  updates to 04_FBP_CGLS_SIRT
   * updates to use different cmaps
   * show noisy CGLS
   * added line profiles

- gvxr recon example

- Update walnut with islicer (https://github.com/TomographicImaging/CIL-Demos/pull/105)

### CODE REVIEW

File: demos/1_Introduction/02_intro_sandstone_parallel_roi.ipynb [hunk 1]
```diff
@@ -413,7 +416,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "With that we are ready to do an FBP reconstruction:"
+    "We set up and run the FBP reconstructor with our dataset and specify the backend to target:"
```
Comment (jakobsj) [inline demos/1_Introduction/02_intro_sandstone_parallel_roi.ipynb]:
```suggestion
    "We set up and run the FBP reconstructor with our dataset and specify the backend to use:"
```

File: demos/1_Introduction/04_FBP_CGLS_SIRT.ipynb [hunk 2]
```diff
@@ -489,30 +508,58 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can see that after iteration 6, the reconstruction gets increasingly more noisy even though the objective value keeps decreasing. After iteration 10, you cannot see significant changes in the reconstruction result."
+    "You can see that after iteration 6, the reconstruction gets increasingly more noisy even though the objective value keeps decreasing. After iteration 10, you cannot see significant changes in the reconstruction result.\n",
+    "\n",
+    "Le us re-run CGLS, this time we will stop at iteration 6 and compare this solution with other alogirithms."
```
Comment (jakobsj) [inline demos/1_Introduction/04_FBP_CGLS_SIRT.ipynb]:
```suggestion
    "Let us re-run CGLS, this time we will stop at iteration 6 and compare this solution with other algorithms."
```

File: demos/1_Introduction/exercises/01_intro_seeds_conebeam.ipynb [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+   "metadata": {},
+   "source": [
+    "Have a go at restricting the size of the image geometry to remove a significant amount of empty space from the reconstruction. Perform the reconstruction using your new image geometry, and display the result using `islicer`.\n",
+    "\n",
+    "1. Create a default `ImageGeometry`\n",
+    "2. Modify the number of voxels to reconstruct\n",
+    "3. Create and run the FDK algorithm\n",
+    "4. Show the reconstructed volume using `islicer`\n",
+    "\n",
+    "00 should provide help in doing this.\n",
```
Comment (jakobsj) [inline demos/1_Introduction/exercises/01_intro_seeds_conebeam.ipynb]: What is 00? Please update to clarify

File: demos/1_Introduction/exercises/01_intro_seeds_conebeam.ipynb [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+   "source": [
+    "..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2090104c-cfbf-495b-a370-abc1a4672fa3",
+   "metadata": {},
+   "source": [
+    "Uncomment the following line to see the solution:"
```
Comment (jakobsj) [inline demos/1_Introduction/exercises/01_intro_seeds_conebeam.ipynb]: Maybe change to "Uncomment and run..." for all these.

File: demos/1_Introduction/exercises/snippets/01_exB.py [hunk 5]
```diff
@@ -0,0 +1 @@
+islicer(data_in, direction='angle',size=10, origin='upper')
```
Comment (jakobsj) [inline demos/1_Introduction/exercises/snippets/01_exB.py]:
```suggestion
islicer(data_in, direction='angle', size=10, origin='upper')
```

**Review by jakobsj — APPROVED** [review]
Looked through quickly, nice improvements and clarifications. Really like the snippets. Just a few typos found, otherwise good to go.

File: demos/1_Introduction/exercises/01_intro_seeds_conebeam.ipynb (hunk 3 above)
Comment (lauramurgatroyd) [reply demos/1_Introduction/exercises/01_intro_seeds_conebeam.ipynb]: I had intended to put a link to notebook 00_CIL_Geometry and I forgot to go back and do it! Thanks!

**Comment by lauramurgatroyd** [comment]
Thanks for the review @jakobsj, I have made the suggested changes 👍

**Pull request merged**